package.json

```diff
@@ -61,7 +68,7 @@
     ],
     "extends": "eslint:recommended",
     "parserOptions": {
-      "ecmaVersion": 2018,
+      "ecmaVersion": 2023,
```

Comment:
at this point we may as well use "latest" or just nothing (default=latest)
so we won't have to update it again